From 629ee84ad0237d1b034c033cd044005c4f0f7778 Mon Sep 17 00:00:00 2001
From: Peng Liu <liupeng@radxa.com>
Date: Mon, 19 Oct 2026 16:37:06 +0800
Subject: [PATCH] arm64: configs: add Radxa desktop configuration fragment

Disable the DRM ioctl permission bypass for the desktop graphics stack.

Signed-off-by: Peng Liu <liupeng@radxa.com>
---
 src/arch/arm64/configs/radxa_custom.config | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 src/arch/arm64/configs/radxa_custom.config

diff --git a/src/arch/arm64/configs/radxa_custom.config b/src/arch/arm64/configs/radxa_custom.config
new file mode 100644
index 000000000000..b54dff5a62dd
--- /dev/null
+++ b/src/arch/arm64/configs/radxa_custom.config
@@ -0,0 +1,2 @@
+# Use standard DRM permissions for the desktop graphics stack.
+CONFIG_DRM_IGNORE_IOTCL_PERMIT=n
